core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -69,7 +69,7 @@ class Partition(val topic: String,
    * In addition to the leader, the controller can also send the epoch of the controller that elected the leader for
    * each partition. */
   private var controllerEpoch: Int = KafkaController.InitialControllerEpoch - 1
-  this.logIdent = "Partition [%s,%d] on broker %d: ".format(topic, partitionId, localBrokerId)
+  this.logIdent = s"Partition $topicPartition on broker $localBrokerId: "
```

Comment:
Shall we use the convention we've been using for `LogContext`? That is `[Foo key=value] `.